forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/model/openbanking/frequency/FrequencyType.java

```diff
@@ -27,7 +27,8 @@
     EVERYWORKINGDAY("EvryWorkgDay"),
     INTERVALWEEKDAY("IntrvlWkDay", "0?([1-9]):0?([1-7])$"),
     WEEKINMONTHDAY("WkInMnthDay", "0?([1-5]):0?([1-7])$"),
-    INTERVALMONTHDAY("IntrvlMnthDay", "(0?[1-6]|12|24):(-0?[1-5]|[1][0-9]|2[0-8]|0?[1-9])$"),
+    INTERVALMONTHDAY("IntrvlMnthDay", "(0?[1-6]|12|24):(-0?[1-5]|0?[1-9]|[12][0-9]|3[01])$"),
+    //"(0?[1-6]|12|24):(-0?[1-5]|[1][0-9]|2[0-8]|0?[1-9])$"),
```

Comment:
You can remove this - it'll be in the git history if we want it.

Reply:
Yes I forgot remove it.

Reply:
done.